### PR TITLE
GH-2201 - Export seed phrase screen adjustment

### PR DESCRIPTION
**Pull request by biafra23**
Handles #2201 

Changes proposed in this pull request:
- Add labels on top and bottom
- redesign word list

New  | Design
----- | --------
image | image

### CODE REVIEW

File: Multisig/UI/UI Library/Views/WarningView.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ class WarningView: UINibView {
         backgroundColor = .backgroundTetriary
 
         titleLabel.setStyle(.headline)
-        descriptionLabel.setStyle(.tertiary)
+        descriptionLabel.setStyle(.secondary)
```
Comment (biafra23) [inline Multisig/UI/UI Library/Views/WarningView.swift:24]: @MouazAlzahabi I think it needs to be secondary?

**Review by DmitryBespalov — COMMENTED** [review]
Looks good! 🚀

File: Multisig/UI/UI Library/Views/WarningView.xib [hunk 2]
```diff
@@ -41,7 +41,7 @@
                                     <nil key="textColor"/>
                                     <nil key="highlightedColor"/>
                                 </label>
-                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="1M0-X0-u76">
+                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="1M0-X0-u76">
```
Comment (biafra23) [inline Multisig/UI/UI Library/Views/WarningView.xib:44]: @MouazAlzahabi Multiline

**Review by DmitryBespalov — APPROVED** [review]
Looks good! 🚀

**Review by elgatovital — APPROVED** [review]

**Pull request merged**